Opensourcecommunity/Contributing-to-the-code.md

```diff
@@ -34,6 +34,6 @@ The GitFlow work flow is someone complicated, but it has the advantage that it g
 
 ### Pull requests: review process
 
-Anyone can send in a pull request. Once you send in a pull request, our developers will check your commit and test it. You can view the SonarCube test results at (http://ci.opensmartgridplatform.org/sonarqube/) and the Jenkins continuous integration results at (http://ci.opensmartgridplatform.org) 
+Anyone can send in a pull request. Once you send in a pull request, assign a maintainer to accept/evaluate your pull request. You can view the SonarCube test results at (http://ci.opensmartgridplatform.org/sonarqube/) and the Jenkins continuous integration results at (http://ci.opensmartgridplatform.org) 
```

Comment:
Anyone can send in a pull request. Once you send in a pull request, assign a maintainer to accept/evaluate your pull request.
 -> Anyone van create a pull request. Then you can assign a maintainer to review your pull request.